content/faqs/Is-Privacy-Badger-spying-on-me.md

```diff
@@ -0,0 +1,10 @@
+---
+question: Why does Privacy Badger need access to my data for all websites?
+weight: 25
+---
+
+When you install Privacy Badger, your browser warns that Privacy Badger can "access your data for all websites" (in Firefox, or "read and change all your data on the websites you visit" in Chrome). You are right to be alarmed. You should only install extensions made by organizations you trust.
+
+Privacy Badger requires these permissions to do its job of automatically detecting and blocking trackers on all websites you visit. We are not ironically (or unironically) spying on you. For more information, see our [Privacy Badger extension permissions explainer](https://github.com/EFForg/privacybadger/blob/master/doc/permissions.md).
+
+Note that the extension permissions warnings only cover what the extension has access to, not what the extension actually does with what it has access to (such as whether the extension secretly uploads your browsing data to its servers). Privacy Badger will never share data about your browsing unless you choose to share it. For more information, see the [EFF privacy policy for software and technology projects](https://www.eff.org/code/privacy/policy).
```

Comment:
as an avid reader of privacy policies, the "unless you choose to share it" is a bit concerning. can we be more specific? like "unless you choose to file a bug report. Even then, EFF will never collect personally-identifying information."

Reply:
Good idea. Unfortunately we use [the same wording on the new user welcome page](https://github.com/EFForg/privacybadger/blob/12d8536d9fd46470d46e08efcdabcda59e346819/src/_locales/en_US/messages.json#L426) ...

Reply:
We should probably update that language as well, good catch.

Reply:
I made this more specific in19cf662d74584b837251c8f6688178d54808d7a2